apps/web-app/pages/[orgShortcode]/convo/[id].vue

```diff
@@ -454,7 +454,7 @@
                 :org-shortcode="orgShortcode">
                 <template #default="{ openFileDialog, loading }">
                   <UnUiButton
-                    label="Upload more"
+                    label=""
```

Comment:
why label removed?
if we want just icon, should use square on the component

Reply:
Yeah, I want a icon only

Reply:
ok, remove label=""
add attribute `square`